overthebox/files/usr/share/otb/post-tracking.d/get-ip-asn

```diff
@@ -26,14 +38,12 @@ for url in $url_list; do
 done
 [ -z "$pub_ip" ] && exit 1
 
-pub_ip_old=$(ubus call "network.interface.$OTB_TRACKER_INTERFACE" status | jq -r '.data.public_ip')
+pub_ip_old=$(otb_get_data "$OTB_TRACKER_INTERFACE/public_ip")
+otb_set_data "$OTB_TRACKER_INTERFACE/public_ip" "$pub_ip"
```

Comment:
after the check

Reply:
Nope, I need to update the modification time of the file. Maybe I could `touch` it instead...